sass/layout/_post.scss

```diff
@@ -1849,7 +1849,7 @@
     .card-icon__container {
         @extend %btn-transition;
         @include opacity(.24);
-        margin-left: 7px;
+        margin: 0 6px 0 4px;
```

Comment:
@asaadmahmood This is causing cherry-pick conflict into release-5.13 since `card-icon__container` class is only applied on master (RHS info cards). I'm curious if this change is related to the main issue - "Bot tag is misaligned in search results for bot accounts".

Reply:
Fixed conflict, included such class into the release branch.

Reply:
Cool.